### ISSUE TITLE
Request - [`panel-primary`]: [`use theme colouring`]

### ISSUE THREAD

**Issue by ldap4life** [wet-boew/GCWeb#1952]
**Is your feature request related to a problem? Please describe.**

The contextual colours for panel's dont look to be overridden. I expected panel-primary to be the gc dark blue, but i think its still the bootstrap original light blue. It would be nice if these were overridden like btn-primary et al.

**Describe alternatives you've considered**

This is possible to accomplish with custom css.

**Comment by duboisp** [comment]
Thanks @ldap4life we will share you suggestion with DTO

\cc @delisma

**Comment by delisma** [comment]
@ldap4life I agree. I'm thinking of a solution that will allow us to pass our design variables to GCWeb. We have the same issue with typography and spacing. So it could be a while before I get to colours, but the intention is to fix this in a systematic manner.
